frappe/website/web_template/section_with_cards/section_with_cards.html

```diff
@@ -11,6 +11,11 @@
 }) -%}
 <div class="mt-6 {{ col_classes }}">
 	<div class="card {{ card_classes }} h-100">
+	        {%- if image -%}
+	        {{ frappe.render_template('templates/includes/image_with_blur.html', {
+			"src": image,
+		}) }}
```

Comment:
The official bootstrap card example uses a flushed image with `card-img-top` class. Any reason not to use that?

Reply:
Hi, applying the class wasn't making any difference as the image is full width and on top anyway. So I decided to remove it.